deployment/base/crdtmerge-deployment.yaml

```diff
@@ -0,0 +1,144 @@
+# https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
+apiVersion: v1
+kind: Service
+metadata:
+  name: crdtmerge
+  namespace: languagedepot
+  labels:
+    app: crdtmerge
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    app: crdtmerge
+  ports:
+  - name: http
+    protocol: TCP
+    port: 80
+
+---
+
+# https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: crdtmerge
+  namespace: languagedepot
+  labels:
+    app: crdtmerge
+spec:
+  selector:
+    matchLabels:
+      app: crdtmerge
+  strategy:
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    # https://kubernetes.io/docs/concepts/workloads/pods/#pod-templates
+    metadata:
+      labels:
+        app: crdtmerge
+    spec:
+      securityContext:
+        runAsGroup: 33
+        runAsUser: 33
+        runAsNonRoot: true
+      containers:
+      - name: crdtmerge
+        image: ghcr.io/sillsdev/lexbox-crdtmerge:develop
+        imagePullPolicy: IfNotPresent
+        # https://kubernetes.io/docs/concepts/configuration/manage-resources-containers
+        resources:
+          requests:
+            memory: 1500Mi
+          limits:
+            memory: 2400Mi
+        startupProbe:
+          httpGet:
+            port: 80
+            path: /api/healthz
```

Comment:
I don't think there's a health endpoint defined so this probably won't work

Reply:
Added a health endpoint in commit 5b2f33b4. I should decide whether I'm using `/api` or not as a prefix, and either make this `/healthz` or make the sync endpoint `/api/sync`. But now the health endpoint should have something listening to it, and we can add IHealthCheck implementations later if we want more detailed checks than the basic "yes, we can respond to HTTP" one.